src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java

```diff
@@ -147,7 +147,8 @@ public void resetProfile() {
   }
 
   @Override
-  public void reZero() {
-    motorOffset = -getAbsoluteMotorPosition();
+  public void setMotorZero() {
+    motorOffset =
```

Comment:
What was wrong with the old code? Why do you need to do unit conversion here?

The unit conversion should ideally just happen inside the motor controller. The more you can talk in the final units (here height in meters), I think it's ideal.

Are you able to set SensorToMechanismRatio in the init? https://www.chiefdelphi.com/t/best-practice-for-inverting-sensor-readings-on-a-talonfx/470876/7

Reply:
Made issue https://github.com/AEMBOT/FRC_2025/issues/66